qtconsole/completion_widget.py

```diff
@@ -20,9 +21,21 @@ def __init__(self, console_widget):
         super(CompletionWidget, self).__init__()
 
         self._text_edit = text_edit
+        self.setEditTriggers(QtGui.QAbstractItemView.NoEditTriggers)
+        self.setHorizontalScrollBarPolicy(QtCore.Qt.ScrollBarAlwaysOff)
+        self.setSelectionBehavior(QtGui.QAbstractItemView.SelectRows)
+        self.setSelectionMode(QtGui.QAbstractItemView.SingleSelection)
+
+        if os.name == 'nt':
```

Comment:
Pushed comments. If there's someone on OSX that can test this in time for the appropriate milestone, then all the better, if not, it does at least fix an existing issue for Windows users.